backend/environment/env.account-service.template

```diff
@@ -1,6 +1,9 @@
 # Copy this file to "<project_root>/.env.account-service", and then tweak values for local development
 # Values in this file will override the same-named environnent variables in `.env.common.docker` for the account-service
 
+# Blockchain node address for the SiwF UI (must be resolvable from a browser)
+FREQUENCY_HTTP_URL=http://localhost:9944
+
 # Base URL for provider webhook endpoints
 # PROVIDER_BASE_URL=https://some-provider/api/v1.0.0
```

Comment:
nit: I think this needs to be correct for account service to work properly, and if we have it in the template, one less thing to worry about correcting.
```suggestion
# PROVIDER_BASE_URL=http://social-app-template-backend:3001/webhooks/account-service
```